web/pandas/pdeps/0014-string-dtype.md

```diff
@@ -215,31 +225,33 @@ and NaN semantics.
-| `StringDtype("python")`                  | `StringDtype(storage="python", na_value=np.nan)`              | "string"                              | (2)      |
-| `StringDtype("pyarrow", na_value=pd.NA)` | `StringDtype(storage="pyarrow", na_value=pd.NA)`              | "string[pyarrow]"                     |          |
-| `StringDtype("python", na_value=pd.NA)`  | `StringDtype(storage="python", na_value=pd.NA)`               | "string[python]"                      |          |
-| `StringDtype(na_value=pd.NA)`            | `StringDtype(storage="pyarrow" \| "python", na_value=pd.NA)`  | "string[pyarrow]" or "string[python]" | (1)      |
-| `StringDtype("pyarrow_numpy")`           | `StringDtype(storage="pyarrow", na_value=np.nan)`             | "string[pyarrow_numpy]"               | (3)      |
+| User specification                          | Concrete dtype                                                | String alias                          | Note     |
+|---------------------------------------------|---------------------------------------------------------------|---------------------------------------|----------|
+| Unspecified (inference)                     | `StringDtype(storage="pyarrow"\|"python", na_value=np.nan)`   | "str"                                 | (1)      |
+| `"str"` or `StringDtype(na_value=np.nan)`   | `StringDtype(storage="pyarrow"\|"python", na_value=np.nan)`   | "str"                                 | (1)      |
+| `StringDtype("pyarrow", na_value=np.nan)`   | `StringDtype(storage="pyarrow", na_value=np.nan)`             | "str"                                 |          |
+| `StringDtype("python", na_value=np.nan)`    | `StringDtype(storage="python", na_value=np.nan)`              | "str"                                 |          |
+| `StringDtype("pyarrow")`                    | `StringDtype(storage="pyarrow", na_value=pd.NA)`              | "string[pyarrow]"                     |          |
+| `StringDtype("python")`                     | `StringDtype(storage="python", na_value=pd.NA)`               | "string[python]"                      |          |
+| `"string"` or `StringDtype()`               | `StringDtype(storage="pyarrow"\|"python", na_value=pd.NA)`    | "string[pyarrow]" or "string[python]" | (1)      |
+| `StringDtype("pyarrow_numpy")`              | `StringDtype(storage="pyarrow", na_value=np.nan)`             | "string[pyarrow_numpy]"               | (2)      |
 
 Notes:
 
 - (1) You get "pyarrow" or "python" depending on pyarrow being installed.
-- (2) Those three rows are backwards incompatible (i.e. they work now but give
-  the NA-variant), see the "Backward compatibility" section below.
-- (3) "pyarrow_numpy" is kept temporarily because this is already in a released
+- (2) "pyarrow_numpy" is kept temporarily because this is already in a released
   version, but we can deprecate it in 2.x and have it removed for 3.0.
```

Comment:
I think the deprecation comment here should be more definitive, i.e., "this PDEP proposes to deprecate it in 2.x and have it removed for 3.0"